MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -862,13 +875,6 @@ public final class MagazineLayout: UICollectionViewLayout {
   // Used to provide the model state with the current visible bounds for the sole purpose of
   // supporting pinned headers and footers.
   private var currentVisibleBounds: CGRect {
-    let contentInset: UIEdgeInsets
-    if #available(iOS 11.0, tvOS 11.0, *) {
-      contentInset = currentCollectionView.adjustedContentInset
-    } else {
-      contentInset = currentCollectionView.contentInset
-    }
-
```

Comment:
not needed - I added a computed property for `contentInset` that does this same logic, so we can use that directly